src/main/java/Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task{
+    private String ddl;
```

Comment:
might want to consider using a more descriptive var name? otherwise good job on this class